configs/olcf/job_script.olcf.bash

```diff
@@ -13,19 +13,16 @@
 #PBS -o mpas_analysis.o$PBS_JOBID
 #PBS -e mpas_analysis.e$PBS_JOBID
 
-cd $PBS_O_WORKDIR   # optional, since this is the default behavior
+cd $PBS_O_WORKDIR
 
-export OMP_NUM_THREADS=1
-
-module unload python python/base
```

Comment:
Here, I was only suggesting you remove the `python/base` part, not the whole command.  If someone is loading  another version of python in their `.bashrc` or `.cshrc`, this still might be needed.  But I think it's also fine to leave it out and let individual users figure out what changes they need to make based on their own configuration.  We really can't anticipate everything people might do...